tests/integration/integration_python_test.go

```diff
@@ -1422,3 +1422,36 @@ func TestParameterizedPython(t *testing.T) {
 		},
 	})
 }
+
+//nolint:paralleltest // ProgramTest calls t.Parallel()
```

Comment:
```suggestion
```

We're not using the full `ProgramTest` here, and also we're calling `t.Parallel` below

Reply:
Oops, I was initially using ProgramTest and then changed it.